Shaders/LTCGI.cginc

```diff
@@ -297,6 +297,7 @@ void LTCGI_Contribution(
                         lmd = smoothstep(0.0, LTCGI_SPECULAR_LIGHTMAP_STEP, saturate(lm - LTCGI_LIGHTMAP_CUTOFF));
                 }
                 ltcgi_output diff;
+                diff.color = 0;
```

Comment:
Does it work if you do this in both places too? That might be a bit cleaner, even though it's already cursed compiler territory.
```suggestion
                ltcgi_output diff = (ltcgi_output)0;
```

Reply:
I am not sure if that resolves it due to when I add the fix and successfully compiles it, then try to undo the change, it continues to succeed compiling. So I don't have a project I can test it in currently since I don't know how to consistently repro it.

As long as the compiler things that its fields are initialized at that point it _should_ work, in theory at least. If I run across it again, I'll certainly give that a test to confirm it.